Rakefile

```diff
@@ -120,3 +128,48 @@ namespace :qless do
   end
 end
 
+
+if ruby_meet_expectation("2.0")
```

Comment:
I wanted to cover cases like >= "1.9.3". With weaker than desirable gemfile support I had to change to "2.0", and the code persisted.